.storybook/main.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
     "@storybook/addon-interactions",
     "storybook-addon-next",
   ],
+  staticDirs: ['../public'],
```

Comment:
Not sure this is needed?

Reply:
Without this line, the picture in the corner is broken 

![image](https://user-images.githubusercontent.com/46238123/197005923-0f9edabe-4947-4960-8e3f-37c059ae31b1.png)

Reply:
oh I see, thank you for clarifying 👍